changelogs/head.asciidoc

```diff
@@ -0,0 +1,11 @@
+[[release-notes-head]]
+== APM Server version HEAD
+
+[float]
+==== Added
+- Support more SSL config options for agent/server communication {pull}2224[2224].
+
+[float]
+==== Removed
+
+https://github.com/elastic/apm-server/compare/7.2\...master[View commits]
```

Comment:
The changelog entry should probably go into `7.2` and not `head`.

Reply:
fwiw, on my duped version of this pr, i chosen to remove the head entry in the conflict resolution, with the intention of doing the usual "fix changelogs" PR later on

Reply:
moved to 7.2